vis/xAPISG-CorrectIncorrectPerQuestion.ipynb

```diff
@@ -27,81 +29,80 @@
-    "    alternatives_list = list(alternatives_list)\n",
-    "    trueFalse=correct_incorrect_question (players_info, alternatives_list)\n",
-    "    show_percentage_chart(True,trueFalse, alternatives_list)\n",
-    "    show_percentage_chart(False,trueFalse, alternatives_list)\n",
+    "    alternatives_list = sorted(get_full_list_of_objects(\"alternatives\"))\n",
+    "    out=interactive(correct_incorrect_question,\n",
+    "             playersSelected=playersSelected,\n",
+    "             eyex=eyex,\n",
+    "             order=order,\n",
+    "             alternatives_list_selected=widgets.SelectMultiple(options=alternatives_list,\n",
+    "                                                               value=alternatives_list,\n",
+    "                                            description='Alternatives:',\n",
+    "                                            disabled=False),\n",
+    "             percentagePerAlternative=False,\n",
+    "             orderLabelValues=[(\"Order by alls true and false responses values\",0),\n",
+    "                               (\"Order by true responses values\",\"true\"),\n",
+    "                               (\"Order by false responses values\",\"false\")])\n",
+    "    md_text=widgets.HTML(\"Bar chart with number of correct and incorrect answers per question (alternative)\")\n",
+    "    return widgets.VBox([md_text,out])\n",
     "\n",
-    "def correct_incorrect_question(players_info, alternatives_list):\n",
+    "def correct_incorrect_question(playersSelected,eyex, order, alternatives_list_selected,percentagePerAlternative, orderLabelValues):\n",
     "    # alternatives correct and incorrect responses\n",
-    "    true_per_alternative = []\n",
-    "    false_per_alternative = []\n",
+    "    true_false_per_alternative = {}\n",
     "    alt_max=0\n",
-    "    for alt in alternatives_list:\n",
+    "    for alt in alternatives_list_selected:\n",
+    "        true_false_per_alternative[alt] = {}\n",
     "        alt_true = 0\n",
     "        alt_false = 0\n",
-    "        for player in players_info.keys():\n",
+    "        for player in playersSelected:\n",
     "             if alt in players_info[player][\"alternatives\"].keys():\n",
     "                _, results = zip(*players_info[player][\"alternatives\"][alt])\n",
-    "                alt_true += results.count(\"True\") +results.count(True) \n",
+    "                alt_true += results.count(\"True\") +results.count(True)\n",
     "                alt_false += results.count(\"False\") +results.count(False)\n",
     "                if alt_true+alt_false>alt_max:\n",
     "                    alt_max=alt_true+alt_false\n",
-    "        true_per_alternative.append(alt_true)\n",
-    "        false_per_alternative.append(alt_false)\n",
+    "        true_false_per_alternative[alt][\"true\"]=alt_true\n",
+    "        true_false_per_alternative[alt][\"false\"]=alt_false\n",
     "    alt_max+=0.5\n",
```

Comment:
same as previous comment

Reply:
I put all in a dictionary to get in memory the association alternative, (true, false) response and the value number to order all after with function "orderDict"